test/fixtures/model/properties.json

```diff
@@ -122,6 +129,25 @@
           "name": "any",
           "type": "Base",
           "isMany": true
+        },
```

Comment:
This does not seem to be used anymore.

Reply:
Not sure, what you are referring to. Property `Root:any` or `Root:any.isMany`?

`Root:any` is used, but I explicitly added property `Root:many` with a fundamental data type to simplify testing. As it's not needed, we can remove `Root:any.isMany` for `Root:any`.